src/client.rs

```diff
@@ -52,14 +53,18 @@ impl Client {
     pub fn new() -> ::Result<Client> {
         let mut client = try_!(new_hyper_client());
         client.set_redirect_policy(::hyper::client::RedirectPolicy::FollowNone);
-        Ok(Client {
-            inner: Arc::new(ClientRef {
-                hyper: RwLock::new(client),
-                redirect_policy: Mutex::new(RedirectPolicy::default()),
-                auto_referer: AtomicBool::new(true),
-                auto_ungzip: AtomicBool::new(true),
-            }),
-        })
+        Ok(
+            Client {
```

Comment:
I can't decide if I should bother to open an issue in rustfmt about this case. I think function calls probably shouldn't be separated like this, but perhaps there's cases where they should be...